packages/fxa-auth-server/test/scripts/bulk-mailer.js

```diff
@@ -54,13 +55,13 @@ function createAccount(email, uid, locale = 'en') {
 }
 
 const account1Mock = createAccount(
-  'user1@test.com',
-  'f9916686c226415abd06ae550f073cec',
+  `user${Math.random()}@test.com`,
+  `${crypto.randomUUID().replaceAll('-','')}`,
   'en'
 );
 const account2Mock = createAccount(
-  'user2@test.com',
-  'f9916686c226415abd06ae550f073ced',
+  `user${Math.random()}@test.com`,
+  `${crypto.randomUUID().replaceAll('-','')}`,
```

Comment:
nit: since this seems to be a copy-pasta of L58-59 above, should we turn these into helper functions? Or create some `createMockAccount()` function that proxies `createAccount()`?

Also, I think you can just use `crypto.randomUUID().replaceAll('-', '')` here without the template string wrapper.